blosc/blosc.c

```diff
@@ -1540,10 +1552,11 @@ int blosc_getitem(const void *src, int start, int nitems, void *dest)
       cbytes = bsize2;
     }
     else {
-      struct blosc_context context;
-      /* blosc_d only uses typesize and flags */
+      struct blosc_context context = {0};
```

Comment:
+1 Nice way to initialize struct to 0.

Reply:
Thanks! It is makes the code slightly slower, but so much easier to debug!